src/graphql/fanout.ts

```diff
@@ -302,6 +300,7 @@ export const DisperseFanout = mutationField('disperseFanout', {
           mint: NATIVE_MINT,
         });
       } catch (e) {
+        console.log(e);
```

Comment:
this looks like production code... should it be using the logger instead of console.log?

Reply:
no logger set up here, will just remove it for now.